packages/esm-appointments-app/__mocks__/appointments.mock.ts

```diff
@@ -0,0 +1,243 @@
+      appointmentNumber: '0000',
+      patient: { identifier: '100GEJ', name: 'John Wilson', uuid: '8673ee4f-e2ab-4077-ba55-4980f408773e' },
+      service: {
+        appointmentServiceId: 1,
+        name: 'Outpatient',
+        description: null,
+        speciality: {},
+        startTime: '',
+        endTime: '',
+        maxAppointmentsLimit: null,
+        durationMins: null,
+        location: {},
+        uuid: 'e2ec9cf0-ec38-4d2b-af6c-59c82fa30b90',
+        color: '#006400',
+        initialAppointmentStatus: 'Scheduled',
+        creatorName: null,
+      },
+      serviceType: {
+        display: 'HIV Clinic',
+        uuid: '53d58ff1-0c45-4e2e-9bd2-9cc826cb46e1',
+        duration: 15,
+      },
+      provider: null,
+      location: { name: 'HIV Clinic', uuid: '2131aff8-2e2a-480a-b7ab-4ac53250262b' },
+      startDateTime: 1631605800000,
+      endDateTime: 1631606100000,
+      appointmentKind: 'WalkIn',
+      status: 'Scheduled',
+      comments: 'Some additional notes',
+      additionalInfo: null,
+      providers: [],
+      recurring: false,
+    },
+    {
+      uuid: '45dcc19d-dd14-4a07-95c6-afa264972a34',
+      appointmentNumber: '0000',
+      patient: { identifier: '100GEJ', name: 'John Wilson', uuid: '8673ee4f-e2ab-4077-ba55-4980f408773e' },
+      service: {
+        appointmentServiceId: 1,
+        name: 'Outpatient',
+        description: null,
+        speciality: {},
+        startTime: '',
+        endTime: '',
+        maxAppointmentsLimit: null,
+        durationMins: null,
+        location: {},
+        uuid: 'e2ec9cf0-ec38-4d2b-af6c-59c82fa30b90',
+        color: '#006400',
+        initialAppointmentStatus: 'Scheduled',
+        creatorName: null,
+      },
+      serviceType: {
+        display: 'TB Clinic',
+        uuid: '4a228e52-0bfe-11ed-861d-0242ac120002',
+        duration: 15,
+      },
+      provider: {
+        uuid: 'f9badd80-ab76-11e2-9e96-0800200c9a66',
+        person: { uuid: '24252571-dd5a-11e6-9d9c-0242ac150002', display: 'Dr James Cook' },
+      },
+      location: { name: 'TB Clinic', uuid: '2131aff8-2e2a-480a-b7ab-4ac53250262b' },
+      startDateTime: 1631623800000,
+      endDateTime: 1631624160000,
+      appointmentKind: 'WalkIn',
+      status: 'Scheduled',
+      comments: 'Some additional notes',
+      additionalInfo: null,
+      providers: [],
+      recurring: false,
+    },
+    {
+      uuid: 'fa4657ad-db46-487d-8e2e-a3858c906ae6',
+      appointmentNumber: '0000',
+      patient: { identifier: '100GEJ', name: 'John Wilson', uuid: '8673ee4f-e2ab-4077-ba55-4980f408773e' },
+      service: {
+        appointmentServiceId: 1,
+        name: 'Outpatient',
+        description: null,
+        speciality: {},
+        startTime: '',
+        endTime: '',
+        maxAppointmentsLimit: null,
+        durationMins: null,
+        location: {},
+        uuid: 'e2ec9cf0-ec38-4d2b-af6c-59c82fa30b90',
+        color: '#006400',
```

Comment:
I think for the colors we need to move this to the color style guide. I doubt if we will need to return the color through api endpoints.